static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -595,7 +595,21 @@ const spanOperationRelativeBreakdownRenderer = (
         }
         return (
           <div key={operationName} style={{width: toPercent(widthPercentage || 0)}}>
-            <Tooltip title={<div>{operationName}</div>} containerDisplayMode="block">
+            <Tooltip
+              title={
+                <div>
+                  <div>{`${operationName} ${formatPercentage(widthPercentage, 0)}`}</div>
+                  <div>
```

Comment:
Not sure you need a wrapping div? If you do, nevermind.

Reply:
Hey Kevan, using div to separate the tooltip text into 2 lines